eventuate-crdt-pure/src/it/scala/com/rbmhtechnology/eventuate/crdt/CRDTChaosSpecLeveldb.scala

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright 2015 - 2016 Red Bull Media House GmbH <http://www.redbullmediahouse.com> - all rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.rbmhtechnology.eventuate.crdt.pure
+
+import java.util.concurrent.ThreadLocalRandom
+import java.util.concurrent.atomic.AtomicInteger
+
+import akka.actor.Props
+import akka.testkit.TestProbe
+import com.rbmhtechnology.eventuate._
+import com.rbmhtechnology.eventuate.crdt.pure.AWSetService.AWSet
+import com.rbmhtechnology.eventuate.crdt.pure.CRDTService.ValueUpdated
+import com.rbmhtechnology.eventuate.crdt.pure.CRDTTypes.Operation
+import com.rbmhtechnology.eventuate.log._
+import com.rbmhtechnology.eventuate.log.leveldb._
+import com.typesafe.config.ConfigFactory
+import org.scalatest._
+
+import scala.collection.immutable.Seq
+import scala.concurrent.{ ExecutionContext, Future }
+
+object CRDTChaosSpecLeveldb {
+  val crdtId = "1"
+
+  def randomNr(): String =
+    ThreadLocalRandom.current.nextInt(1, 10).toString
+}
+
+class CRDTChaosSpecLeveldb extends WordSpec with Matchers with MultiLocationSpecLeveldb {
+  import ReplicationIntegrationSpec.replicationConnection
+  import CRDTChaosSpecLeveldb._
+
+  class TestEventLog(id: String) extends LeveldbEventLog(id, "log-test") {
+    override def write(events: Seq[DurableEvent], partition: Long, clock: EventLogClock): Unit =
+      if (events.map(_.payload).contains(ValueUpdated(AddOp(randomNr())))) throw IntegrationTestException else super.write(events, partition, clock)
+  }
+
+  // FIXME allow use write-batch-size > 1
```

Comment:
Is that still open? Do you still plan to fix this?

Reply:
I've tried without success with 
```
class TestEventLog(id: String) extends LeveldbEventLog(id, "log-test") {
    override def write(events: Seq[DurableEvent], partition: Long, clock: EventLogClock): Unit = {
      val containsStop = events.map(_.payload).exists {
        case ValueUpdated(AddOp(s)) => s.toString.startsWith("stop")
        case _                      => false
      }
      if (!containsStop && events.map(_.payload).contains(ValueUpdated(AddOp(randomNr())))) throw IntegrationTestException else super.write(events, partition, clock)
    }
  }
```
I can't see what I'm missing, so I could let the `FIXME` as a reminder or delete it.

Reply:
For me both is fine.